collections/_article/development-fund-perks.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "New Dev Fund Perks: Discord Role and Forum Badge"
+excerpt: "We are finally unveiling the new Godot forum which replaces our Q&A platform while unlocking many new features."
+categories: ["news"]
+author: Winston Yallow
+image: /storage/blog/covers/development-fund-perks.webp
+date: 2024-07-10 12:00:00
+---
+
+We have added new benefits for members of the godot development fund! On our forum, a special Badge is now available as a thanks for your continued support. And you can also get a custom role on our new and official [discord server](https://discord.com/invite/7HtykRQ4Rg)! Read the instructions below for connecting your accounts.
+
+Not a fund member yet? [Subscribe now!](https://fund.godotengine.org/#membership)
+
+## Connecting your accounts
+
+### Forum Badge
+
+The forum badges are assigned automatically to your Godot account when you become a fund member.
+
+![Badge with a golden Godot logo and the text "Fund Member - Member of the Development Fund", awarded to 47 users](/storage/blog/development-fund-perks/forum-badge.webp)
+
+Already a fund member but your badge does not show up? Make sure that you have signed in at least once to the [development fund](https://fund.godotengine.org/#membership). It may take up to 24 hours for the badge to appear on the forum.
```

Comment:
```suggestion
Already a Development Fund member but your badge does not show up? Make sure that you have signed in at least once to the [development fund](https://fund.godotengine.org/#membership). It may take up to 24 hours for the badge to appear on the forum.
```